src/qibosoq/qick_programs.py

```diff
@@ -174,12 +184,15 @@ def declare_gen_mux_ro(self):
                 adc_ch_added.append(adc_ch)
                 # TODO add parameters to QickProgramConfig
                 freq = pulse.frequency - self.mixer_freq - self.LO_freq
-                zone = 1 if freq < self.max_sampling_rate / 2 else 2
+                max_sampling_rate = 1_536_000_000  # TODO this should be in config
```

Comment:
See above (and considering that this appears twice, you can also immediately reuse the constant)